dianna/methods/lime_timeseries.py

```diff
@@ -117,15 +116,15 @@ def explain(
             )
         # extract scores from lime explainer
         saliency = []
-        for i, _ in enumerate(labels):
-            local_exp = sorted(exp.local_exp[i])
+        for i, label in enumerate(labels):
+            local_exp = sorted(exp.local_exp[label])
             # shape of local_exp [(index, saliency)]
             selected_saliency = [i[1] for i in local_exp]
             saliency.append(selected_saliency[:])
 
         return np.concatenate(saliency).reshape(-1, sequence, n_var)
 
-    def _calculate_distance(self, masked_data, distance_method="cosine"):
+    def _calculate_distance(self, masked_data, distance_method='cosine'):
```

Comment:
Just notice that your formatter prefers single quote. I use `black` and it turns everything to double quote. Maybe we can create an issue and put it in the developer doc about which formatter we prefer?

Reply:
I'm pretty sure we already have this: https://dianna.readthedocs.io/en/latest/developer_info.html#running-linters-locally 😅